en/get-started.md

```diff
@@ -496,10 +496,10 @@ APPROXIMATE_KEYS: 0
 ```sql
 mysql> select tidb_version()\G
 *************************** 1. row ***************************
-  tidb_version(): Release Version: v6.5.0
+  tidb_version(): Release Version: v7.1.0
          Edition: Community
  Git Commit Hash: 4a1b2e9fe5b5afb1068c56de47adb07098d768d6
-      Git Branch: heads/refs/tags/v6.5.0
+      Git Branch: heads/refs/tags/v7.1.0
```

Comment:
it's better to update the above `Git Commit Hash` to the real value after TiDB v7.1.0 is released.

Reply:
Can you please update this now that 7.1 is released? Thank you~